modules/rubiconBidAdapter.js

```diff
@@ -500,14 +500,15 @@ function _getDigiTrustQueryParams() {
  * @param {BidRequest} bidRequest
  * @returns {string}
  */
-function _getPageUrl(bidRequest) {
+function _getPageUrl(bidRequest, bidderRequest) {
   let pageUrl = config.getConfig('pageUrl');
   if (bidRequest.params.referrer) {
     pageUrl = bidRequest.params.referrer;
   } else if (!pageUrl) {
-    pageUrl = utils.getTopWindowUrl();
+    pageUrl = bidderRequest.refererInfo.referer;
   }
-  return bidRequest.params.secure ? pageUrl.replace(/^http:/i, 'https:') : pageUrl;
+  let _pageUrl = bidRequest.params.secure ? pageUrl.replace(/^http:/i, 'https:') : pageUrl;
+  return decodeURIComponent(_pageUrl.replace(/\+/g, ' '))
```

Comment:
the title of the PR says, "DecodeUrl `if` url is encoded".   Do we need to check first if it's encoded?

Reply:
question - should we update the referer module to return unencoded referer? I'm sort of on the fence on this but curious to get other's input.

Reply:
@harpere , I guess that if the url is decoded, decoding should not provoke any error.  

@mkendall07 , I would indeed recommend returning decoding url as other partner may face some issues.

Reply:
@AntoineJac - I think decoding again might only make a difference if there was a `%` in the referrer.   But if we know the module is encoding, we're fine.

@mkendall07 - my impulse is that the module should NOT encoded referer and let the adapter encode it if they need to.   But not a big deal.